network-api/networkapi/wagtailpages/pagemodels/research_hub/detail_page.py

```diff
@@ -64,7 +64,7 @@ class ResearchDetailPage(research_base.ResearchHubBasePage):
 
     content_panels = wagtail_models.Page.content_panels + [
         image_handlers.FieldPanel("cover_image"),
-        edit_handlers.InlinePanel("research_links", heading="Research links", min_num=1),
+        edit_handlers.InlinePanel("research_links", heading="Research links"),
```

Comment:
I did notice when testing that this didn't have a max (which i was going to raise) but I think this change in itself is fine :+1: